src/main/java/se/bjurr/violations/lib/example/OtherClass.java

```diff
@@ -4,12 +4,15 @@
  * No ending dot
```

Comment:
Reporter: PMD
Rule: OverrideBothEqualsAndHashcode
Severity: WARN
File: /home/bjerre/workspace/jenkinsci/violation-comments-to-github-plugin/work/workspace/test/violations-test/src/main/java/se/bjurr/violations/lib/example/OtherClass.java L19
 

Basic http://pmd.sourceforge.net/pmd-5.2.3/pmd-java/rules/java/basic.html#OverrideBothEqualsAndHashcode

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>